apps/coscrad-frontend/src/components/transcripts/transcript-presenter.tsx

```diff
@@ -0,0 +1,42 @@
+import { SubtitlesRounded } from '@mui/icons-material';
```

Comment:
Need to use the standard `import { SubtitlesRounded as SubtitlesRoundedIcon } from '@mui/icons-material';

Reply:
Please alias this as `SubtitlesRoundedIcon`.

Reply:
We did this.